docs/openapi/openapi.yaml

```diff
@@ -189,6 +195,29 @@ components:
               example:
                 - source: "/uid"
                   detail: "invalid uid format"
+    NotFoundError:
+      allOf:
+        - $ref: "#/components/schemas/AbstractError"
+        - type: object
+          properties:
+            code:
+              enum: ["NOT_FOUND"]
+            errors:
+              type: array
+              items:
+                type: object
+                required:
+                  - source
+                  - detail
+                properties:
+                  source:
+                    type: string
+                    format: jsonpointer
+                  detail:
+                    type: string
+              example:
+                - source: "/uid"
+                  detail: "uid not found"
```

Comment:
You can ditch this, since "not found" errors don't include any field-specific error messages
```suggestion
```